clang/lib/AST/ExprConstant.cpp

```diff
@@ -14926,6 +14926,42 @@ bool IntExprEvaluator::VisitUnaryExprOrTypeTraitExpr(
 
     return false;
   }
+  case UETT_CountOf: {
```

Comment:
Did you consider the new interpreter? @tbaederr

Reply:
I'll add support for this there.